### DISCUSSION TITLE
Hiding specific elements of the sidebar

### DISCUSSION THREAD

**Discussion by seanlewis08** [pradyunsg/furo#786] [Questions] [answered]
I have some markdown files in my docs which I am referencing to, but I do not want them to show up in the sidebar contents. I am using hidden in the toctree to hide them from the main table of contents, but I can not figure out how to hide them in the sidebar as they are generated for every single page. Am I missing something here? Is this not possible?

**Comment by pradyunsg** [comment, marked as answer]
You can make those documents `orphan` and exclude them from the toctree. They would be excluded from the sidebar entirely in those cases.

```
---
orphan: true
---
```
